atlasdb-client/src/main/java/com/palantir/atlasdb/offheap/rocksdb/RocksDbPersistentTimestampStore.java

```diff
@@ -68,6 +78,39 @@ public Long get(StoreNamespace storeNamespace, @Nonnull Long startTs) {
         return startTs + (Long) ValueType.VAR_LONG.convertToJava(value, 0);
     }
 
+    @Override
+    public Set<Map.Entry<Long, Long>> multiGet(StoreNamespace storeNamespace, List<Long> keys) {
+        Preconditions.checkArgument(
+                availableColumnFamilies.containsKey(storeNamespace.uniqueName()),
+                "Store namespace does not exist");
```

Comment:
We could refactor this to a separate check?

Reply:
Refactored.